cli/cmd/simulation/opdom.hpp

```diff
@@ -306,6 +319,34 @@ class opdom_command : public command
             env->out() << fmt::format("[e] {}", e.what()) << std::endl;
         }
     }
+    /**
```

Comment:
@wlambooy Thanks for adding this. It is super helpful. However, I prefer the way it is done in https://github.com/cda-tum/fiction/blob/4853c86f0431ad6e3ee6139f34f75e43961e5c4f/cli/cmd/physical_design/exact.hpp#L39C1-L41C9.

Reply:
Thanks, that's better